### DISCUSSION TITLE
Code Assignments for The Erlang Project

### DISCUSSION THREAD

**Discussion by itsclll** [9479Team03ErlangProject/AppDevFinalProject#1] [Announcements]
Hello team,
As part of our Erlang project, we are assigning specific modules to each team member for development. Below are the assignments for the code modules:
@vinny-Kev : user_server.erl
Focus: Develop the user server functionality, handling user data and processes.
@leesnx : chat_app.erl
Focus: Implement the main chat application logic, including managing user interactions.
@JayPing23 : chat_supervisor.erl
Focus: Create the supervisor module for chat processes, ensuring fault tolerance and supervision hierarchy.
@Vanilla: chat_room.erl
Focus: Build the chat room module to manage communication within individual chat rooms.
Please ensure:
    - Code Structure: Follow the established coding standards and guidelines for Erlang.
    - Commit Logs: Use meaningful commit messages that clearly describe the changes made.
    - Testing: Write and run tests for your module before submitting for review.
    - Collaboration: If you have dependencies with another team member's module, coordinate effectively to ensure compatibility.

**Comment by JayPing23** [comment]
I'm done with my part which is the chat_supervisor.erl 
@vinny-Kev you can now start

**Comment by vinny-Kev** [comment]
I have also completed my part which is the user server, the next part to be completed is @Vanilla-afk's work. Thank you!

**Comment by Vanilla-afk** [comment]
I'm done with my part which is the chat room, next part is yours @leesnx . Thank you!!

**Comment by ghost** [comment]
I have completed the chat_app.erl please review our repository. Thank you!
